dpbench/runner.py

```diff
@@ -24,26 +33,23 @@ def run_benchmark(
     timeout=10.0,
 ):
     print("")
-    print("===============================================================")
+    print("================ Benchmark " + bname + " ========================")
     print("")
 
-    bdir = "benchmarks/" + bname
-    if not pkg_resources.resource_isdir(__name__, bdir):
-        return
-    if bname == "__pycache__":
-        return
-    bench = None
     try:
-        bench = dpbi.Benchmark(bname=bname, bconfig_path=bconfig_path)
-    except Exception:
+        benchmod = importlib.import_module("dpbench.benchmarks." + bname)
+        bench = dpbi.Benchmark(benchmod, bconfig_path=bconfig_path)
+    except Exception as e:
+        print(e)
         warnings.warn(
             "WARN: Skipping the benchmark "
             + bname
-            + ". No configuration could not be found."
+            + ". Could not create a Benchmark object."
```

Comment:
It is OK for now. But later maybe can consider more meaningful message.

Reply:
How about: `warnings.warn("Skipping the benchmark execution due to following error: " + e.__str__`?